docs/it-manual/sre-playbook/monitoring.md

```diff
@@ -50,7 +50,11 @@ With Prometheus connected as a Grafana workspace, panels can now be created in G
 
 To import this pre-built dashboard: 
 
-1. Hover over the "+" icon in the left nav
-1. Click the "Import" option 
-1. Paste [the JSON here](https://gist.githubusercontent.com/bion/9c5e9293a887f4ea76f123908fb35ef7/raw/c179075f37026544124475ed46646861fda16a5f/civiform_grafana_dashboard.json) into the "Import via panel JSON" and click "Load"
-1. Fill in the details for the imported dashboard, selecting your CiviForm prometheus instance for the data source
+1. Click on the "Data Sources" tab and select the Prometheus data source
+2. Copy the ID at the end of the URL after `/edit/`
+3. Hover over the "+" icon in the left nav
+4. Click the "Import" option 
+5. Paste [the JSON here](https://gist.githubusercontent.com/dkatzz/204b5c411cb43b8617b777efe2567b01/raw/d146013037c8e1ab8f7a1616f94ab59444170a11/gistfile1.txt) into the "Import via panel JSON" 
+6. Replace all instances of ${PROMETHEUS_ID} with the ID you copied in step 2
```

Comment:
Just curious why you opted for this way instead of creating a variable. Also, did you try it out this way and have success? I feel like we ran into some problems with copy/pasting when we were pairing on it yesterday.

Reply:
The variable worked when you loaded the URL with the &prometheusid=X in the URL, but it didn't work when that wasn't included. I figured since this is a one time setup for each deployment it isn't too bad to do the copy and pasting and easier than having to enter the variable value sometimes. Thoughts? I tried out the copy paste and was able to get it working!

Reply:
awesome! that sounds good to me. that's good to know about the url.